CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.0 (2020-05-11)
```

Comment:
Since this is a new feature, and not a bug fix, I felt it warranted a minor version update. In keeping with [SemVer](https://semver.org/), this update contains no breaking changes.